prov/verbs/src/fi_verbs.h

```diff
@@ -365,8 +343,7 @@ struct vrb_domain {
 
 	ssize_t		(*send_credits)(struct fid_ep *ep, uint64_t credits);
 
-	/* Indicates that MSG endpoints should use the XRC transport.
-	 * TODO: Move selection of XRC/RC to endpoint info from domain */
+	/* Indicates that MSG endpoints should use the XRC transport. */
```

Comment:
This is separate from this change, but if flags is a bool, we should convert it to that, and rename it to indicate what it represents (use_xrc), so we can drop this comment.  If flags is not a bool, then this comment should still go away and be placed where the flag is defined, if that flag name doesn't make it obvious.

Reply:
> This is separate from this change, but if flags is a bool, we should convert it to that, and rename it to indicate what it represents (use_xrc), so we can drop this comment. If flags is not a bool, then this comment should still go away and be placed where the flag is defined, if that flag name doesn't make it obvious.

I agree, I will open create a commit for this outside of this PR. Thanks.

Reply:
@shefty - Looks like the flag field is now used to indicate the domain supports ODP as well. I'm adding a separate change to clean up the comment outside of this PR.